optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -866,6 +868,9 @@ def forward(
             output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
         )
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
+        if self.generation_config.flash_attention_fast_softmax_mode == 'fast':
```

Comment:
Change flash_attention_fast_softmax_mode to flash_attention_fast_softmax & make it a boolean. Also by default it should be set to "False" until the underlying changes to support this mode are integrated to master.